test/e2e/framework.go

```diff
@@ -1683,6 +1683,20 @@ func (data *TestData) PatchPod(namespace, name string, patch []byte) error {
 	return nil
 }
 
+func (data *TestData) GetCRDClient() (crdclientset.Interface, error) {
+	if data.crdClient == nil {
+		return nil, fmt.Errorf("CRD client is not initialized")
+	}
+	return data.crdClient, nil
+}
+
+func (data *TestData) GetKubeConfig() (*restclient.Config, error) {
+	if data.kubeConfig == nil {
+		return nil, fmt.Errorf("kubeconfig is not initialized")
+	}
+	return data.kubeConfig, nil
+}
```

Comment:
I don't think this is needed. All our existing test cases use `data.crdClient` and `data.kubeConfig` directly. It's up to the testing framework to ensure that these are initialized successfully. Am I missing something?

Reply:
Actually i created these getter functions because fields are private I cannot access it directly?

Reply:
you should make the fields that you need public. Actually `KubeConfig` is already 

I have noticed that the SRIOV test creates a new client, using `testData.KubeConfig`, which doesn't make sense to me as we should just be able to use the existing client: https://github.com/antrea-io/antrea/blob/61f55bbf9e9be049162e218a39786a4f41cb08bc/test/e2e-secondary-network/secondary_network_test.go#L339-L342

Anyway, just like `data.KubeConfig` is exported now (you should rebase your PR), we can export `data.ClientSet` and `data.CrdClient`